packages/core/src/orchestrate/call.ts

```diff
@@ -53,28 +58,30 @@ export async function call<Model extends ILlmSchema.Model>(ctx: AgenticaContext<
         {
           role: "system",
           content:
-            ctx.config?.systemPrompt?.execute?.(ctx.histories)
+            (ctx.type === "context"
+              ? ctx.config?.systemPrompt?.execute?.(ctx.histories)
```

Comment:
do we need conditional operator?

Reply:
return value is same!

Reply:
Just for avoiding as statement, but current working code is doing it lol

Reply:
understand, it's good description...
I'm sure it hurts the reader's readability, but I don't know a better way lol